tests/rptest/tests/group_membership_test.py

```diff
@@ -113,3 +116,187 @@ def _check_no_duplicates():
                    10,
                    backoff_sec=2,
                    err_msg="found persistent duplicates in groups listing")
+
+
+class GroupMetricsTest(RedpandaTest):
+    def __init__(self, ctx, *args, **kwargs):
+
+        extra_rp_conf = dict(default_topic_replications=3, )
```

Comment:
this is already the default replication. I don't think you need to set it here.

Reply:
I need it to have replication of `kafka_internal` `group` topic

Reply:
`I need it to have replication of kafka_internal group topic`

what is "it"? you can set replication factor explicitly on topics when you create them. if you mean the kafka internal topic itself, i think that its already defaulted to 3, right?

Reply:
I don't know where to find default value iin tests. But without this config I can't transfer leadership to another node, redpanda returns 500

Reply:
got it. then leave it for now and ill take a look to see what's going on.

Reply:
the topic is created in `try_create_consumer_group_topic` and it does use `default_topic_replication` so this change looks ok. I thought it worked differently :)